modules/docs/mdx/style-props/STYLE_PROPS.mdx

```diff
@@ -37,6 +38,20 @@ import {TextExample} from './examples/Text';
 Style props provide a common, ergonomic API for modifying a component's styles by passing styles
 with props.
 
+<InformationHighlight variant="caution" style={{margin: '1.5rem 0'}}>
+  <InformationHighlight.Icon />
+  <InformationHighlight.Heading>Important</InformationHighlight.Heading>
+  <InformationHighlight.Body>
+    While some of our components still support style props, we have made efforts to migrate away
+    from Emotion due to its performance cost. We strong advise avoid using style props and instead
+    use our new styling utilities. If you need to customize styles, please reference our docs on how
+    to customize styles.
```

Comment:
```suggestion
    While some of our components still support style props, we have made efforts to migrate away
    from Emotion due to its performance cost. We strongly advise to avoid using style props and instead
    use our new styling utilities. If you need to customize styles, please reference our docs on how
    to customize styles.
```

Reply:
Should we link to the docs on how to customize styles?